test/OpenTelemetry.Tests/Trace/ExceptionProcessorTest.cs

```diff
@@ -86,23 +86,23 @@ public void ActivityStatusSetToErrorWhenExceptionProcessorEnabled()
             }
 
             Assert.Equal(StatusCode.Error, activity1.GetStatus().StatusCode);
-            Assert.Null(activity1.GetTagValue("otel.exception_pointers"));
+            Assert.Null(GetTagValue(activity1, "otel.exception_pointers"));
```

Comment:
This was the only conflict in splitting tests.
This test is verifying the `ExceptionProcessor` in the OpenTelemetry project.
This test was using the `GetTagValue` extension method from the OpenTelemetry.Api project.

This extension class has other internal dependencies, so I decided to just copy the method into this test class (see below).

Reply:
> This extension class has other internal dependencies

I think you'll have to deal with this at some point, right? Since in order to remove InternalsVisible to from the OpenTelemetry.Api project to the OpenTelemetry project, this method will probably ultimately move to the shared project.

Though, I'd be ok with copying the method as a temporary thing if the extension class can not be easily exposed to the test project.

Reply:
I'll make a note to circle back to this. Thanks